apps/librelingo_json_export/tests/test_cli.py

```diff
@@ -86,3 +96,18 @@ def test_dry_run_doesnt_create_output_files(
 def test_dry_run_calls_real_export_course(mocks, inputs, invoke):
     invoke([*inputs, "--dry-run"])
     assert mocks["export_course"].call_count == 1
+
+
+@pytest.mark.parametrize("no_dry_run_arg", [["--no-dry-run"], []])
+def test_creates_excpected_files(fs, invoke_with_no_mocks, no_dry_run_arg):
+    input_path = Path(__file__).parent / "fixtures" / "fake_course"
+    fs.add_real_directory(input_path)
+    output_path = fakes.path()
+    assert not output_path.exists()
+    expected_files = {
+        output_path / _
+        for _ in ("courseData.json", "challenges/hello.json", "introduction/hello.md")
+    }
```

Comment:
```suggestion
    expected_files = {
        output_path / "courseData.json",
        output_path / "challenges/hello.json",
        output_path / "introduction/hello.md",
    }
```

No need to put some extra logic for 3 examples

Reply:
The logic is not so complex here in my opinion. I would leave it as it is.

Reply:
it's true that the logic is not so complex but it doesn't read as easily, and does not seem to add any benefit, since we don't expect this list to grow or become dynamic